core/chains/evm/txmgr/orm.go

```diff
@@ -232,6 +233,20 @@ func (o *orm) FindEthTxWithAttempts(etxID int64) (etx EthTx, err error) {
 	return etx, errors.Wrap(err, "FindEthTxWithAttempts failed")
 }
 
+func (o *orm) FindEthTxAttemptConfirmedByEthTxIDs(ids []int64) (etx []EthTxAttempt, err error) {
+	var attempts []EthTxAttempt
+	err = o.q.Transaction(func(tx pg.Queryer) error {
+		if err = tx.Select(&attempts, `SELECT * FROM eth_tx_attempts WHERE eth_tx_id = ANY($1) ORDER BY eth_tx_attempts.gas_price DESC, eth_tx_attempts.gas_tip_cap DESC`, ids); err != nil {
```

Comment:
Could this not find a whole bunch of unconfirmed attempts? Seems to be missing a `JOIN` clause to `eth_receipts`.

Reply:
right, thats a good idea, will do that.